cpu/cc2538/include/periph_cpu.h

```diff
@@ -200,7 +205,7 @@ typedef struct {
  * @{
  */
 typedef struct {
-    cc2538_ssi_t *dev;      /**< SSI device */
+    uint8_t num;            /**< SSI device */
```

Comment:
The semantics of this parameter changed, but the documentation didn't. Can you extend the comment? For example, mention that this is the device **number** 1...x or so